utils/pluginScripts/yarnInstallPlugins.js

```diff
@@ -13,6 +14,10 @@ console.log('running yarn install for plugins')
 const command = 'yarn install'
 plugins.forEach((plugin) => {
   const { fullPluginFolder, pluginName } = plugin
+  if (!fs.existsSync(path.join(fullPluginFolder, 'package.json'))) {
+    console.log('no package.json: ' + pluginName)
+    return
+  }
```

Comment:
When this bug happened for me, it was when there was empty plugin folders (from previously renamed plugins on another branch), so it wasn't just the lack of a `plugin.json`. What I mean is that, will this fix also handle folders within "plugins" that have _nothing_ in them, too?

Reply:
ohh it's looking for `package.json`, that should be enough

Reply:
Sweet. So it never looks for anything else? What about `index.ts`?

Reply:
nope, it just stops it from running 'yarn install' which causes an error to be thrown and something funky happens in `postinstall`, i think every plugin should have `package.json` as it has build instructions for it.

and you've done good checking on application startup for other things (in case of empty folders like this)